llmfoundry/command_utils/data_prep/convert_text_to_mds.py

```diff
@@ -478,7 +478,9 @@ def convert_text_to_mds(
     index_path = os.path.join(local_output_folder, 'index.json')
     with open(index_path, 'r') as index_file:
         if not json.load(index_file)['shards']:
-            raise DatasetTooSmallError()
+            raise DatasetTooSmallError(
```

Comment:
this exception already existed before right? just making sure, for my understanding

Reply:
yup it did